pkg/common/utils/resources.go

```diff
@@ -248,3 +248,11 @@ func (r *Resources) MultiplyBy(i int64) {
 	r.MemoryRequests.Set(r.MemoryRequests.Value() * i)
 	r.Disk.Set(r.Disk.Value() * i)
 }
+
+func (r *Resources) AddByResource(resource Resources) {
```

Comment:
This function name is confusing :)

`AddByResource` seems to imply that you do some addition depending on the type of the resource.

I would probably call it `AddResources` or something, so it's clear that `resources` is added to `r` when you do `r.AddResources(resources)`

Reply:
Fixed